rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -627,7 +627,9 @@ def get_history_events(
             changes = self._process_trove_events(trove['changes'], from_timestamp, to_timestamp)
             result.append(changes)
         # Flatten the result (list of lists to list)
-        return reduce(add, result)
+        if len(result) == 0:
+            return reduce(add, result)
+        return result
```

Comment:
Also the linter is complaining here because it assigns the first possible type that is `List[List[DefiEvent]]` instead of `List[DefiEvent]`.
A way to solve this could be to do

```suggestion
        return []
```